contracts/src/v0.8/vrf/test/VRFV2PlusWrapper_Arbitrum.t.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {BaseTest} from "./BaseTest.t.sol";
+import {MockLinkToken} from "../../mocks/MockLinkToken.sol";
+import {MockV3Aggregator} from "../../tests/MockV3Aggregator.sol";
+import {ExposedVRFCoordinatorV2_5_Arbitrum} from "../dev/testhelpers/ExposedVRFCoordinatorV2_5_Arbitrum.sol";
+import {VRFV2PlusWrapper_Arbitrum} from "../dev/VRFV2PlusWrapper_Arbitrum.sol";
+import {ArbGasInfo} from "../../vendor/@arbitrum/nitro-contracts/src/precompiles/ArbGasInfo.sol";
+
+contract VRFV2PlusWrapperTest is BaseTest {
```

Comment:
Should we name this VRFV2PlusWrapperArbitrumTest?

Reply:
I missed this, thanks. Done!